### PR TITLE
#4665 [Manager] Redirect to workspace settings pages from Toolbox's add buttons

### PR DESCRIPTION
**Pull request by konekoya**
### Checklist
- [x] [Review Contribution Guidelines](https://ohara.readthedocs.io/en/latest/contributing.html)
- [x] Add Reviewers (see [Authors](https://github.com/oharastream/ohara#ohara-team))
- [x] Add Assignees (of course, you are the assignee by default!)
- [x] DON'T set a Milestone
- [x] DON'T add any labels (PR labels will be added automatically by Jenkins)
- [x] Link to an issue(You can do so by using the required by keyword: required by #4665)
- [ ] Add unit tests (or please explain why this PR is not worth having new tests)
- [ ] Pass QA (Sometimes you may encounter an existing flaky test which would obstruct you from getting QA passed. If the failed tests are unrelated, you can add a comment in the PR thread)

### CODE REVIEW

**Comment by konekoya** [comment]
retry manager-e2e

File: ohara-manager/client/src/components/Pipeline/Toolbox/ToolboxList.js [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+  const {
+    toolboxHeight,
+    toolboxBodyRef,
+    panelAddButtonRef,
+    panelSummaryRef,
+    handleScroll,
+    pipelineDispatch,
+    expanded,
+  } = props;
+  const { open: openEditWorkspaceDialog } = context.useEditWorkspaceDialog();
```
Comment (eechih) [inline ohara-manager/client/src/components/Pipeline/Toolbox/ToolboxList.js:42]: @konekoya Can you refact this name by the way? Maybe it is named  `useWorkspaceSettingsDialog`

Comment (konekoya) [reply ohara-manager/client/src/components/Pipeline/Toolbox/ToolboxList.js:42]: Ah, yes. Sorry, I forgot to do so 🤦

Comment (eechih) [reply ohara-manager/client/src/components/Pipeline/Toolbox/ToolboxList.js:42]: Thanks :)

**Comment by konekoya** [comment]
retry manager-e2e

**Pull request merged**